### DISCUSSION TITLE
Is there a function that returns the event occurrence probability between two time points?

### DISCUSSION THREAD

**Discussion by yildizyasin** [CamDavidsonPilon/lifelines#1285] [Q&A] [answered]
I am looking for a function that returns the probability of hazard between two time points for a fitted model. I'll use these probabilities as transition probabilities for a discrete time Markov Chain so I need transition probabilities for hazards.

Looking for: `P(t<=T<t+1|T>=t)` which is equivalent to` (S(t) - S(t+1)) / S(t)` by Bayes' formula.

**Comment by CamDavidsonPilon** [comment, marked as answer]
Hi @yildizyasin, there's no built in function. Are you fitting a regression model? If so, there is the `conditional_after` argument in `predict_survival_function` that could be useful. However, you latter equation can be easily turned into a function, too.

**Reply by yildizyasin** [reply]
Yes I'll use it as a regression model to predict event occurrences between any specific time period. But, I'll use only time(t) parameter for my regression. Namely, I want to use my simple fitted survival distribution to calculate such a probability. 

For now I wrote a simple function that returns what I expected. I may create a pull request when I am available to tidy up the code.

**Reply by yildizyasin** [reply]
btw. not sure if it is a common demand. Maybe it is better to keep it in my local project.

**Reply by CamDavidsonPilon** [reply]
I don't think it's very common (not common enough to have a name), plus it's easy to recreate locally, so I lean towards not adding it natively yet.
